CHANGELOG.rst

```diff
@@ -2,6 +2,22 @@
 Changelog
 =========
 
+Version 1.2.1
```

Comment:
Version 1.3.0 :)

Reply:
Ouups ! 😅 
Fixed in aefda7f4990c201765a24809f80f992ed8d8826d.
Thanks !